src/components/ec-panel/ec-panel.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <aside
+    v-if="showPanel"
+    class="ec-panel"
+  >
+    <header class="ec-panel__header">
+      <div class="ec-panel__header-icons">
+        <ec-icon
+          class="ec-panel__close"
+          name="simple-close"
+          :size="24"
+          @click="$emit('close')"
```

Comment:
You need to emit the new value because the event is used for v-model. Otherwise this will never work:
```
<ec-panel v-model="myProp" />
```
and you always have to pass `<ec-panel v-model="myProp" @close="myProp = !myProp"  />`. That's why is it working in your story but if you drop `@close` handler, it would stop. 

Write a test for a `v-model`.